src/Dazinator.Extensions.Options.Updatable/JsonUpdatableOptions.cs

```diff
@@ -8,68 +8,83 @@ namespace Dazinator.Extensions.Options.Updatable
     public class JsonUpdatableOptions<TOptions> : IUpdatableOptions<TOptions>
         where TOptions : class, new()
     {
-        private readonly IOptionsMonitor<TOptions> _monitor;
+        // ##############################################################################################################################
+        // Properties
+        // ##############################################################################################################################
 
-        // private IOptionsSnapshot<TOptions> _options;
-        private readonly IJsonStreamProvider<TOptions> _jsonFileStreamProvider;
-       // private readonly IOptionsMonitorCache<TOptions> _cache;
-        private readonly string _sectionName;
-        private readonly bool _leaveOpen;
-        private readonly static JsonSerializerOptions _defaultSerializerOptions = new JsonSerializerOptions() { IgnoreNullValues = true, WriteIndented = true };
+        #region Properties
 
-        public TOptions Value => _monitor.CurrentValue;
+        // ##########################################################################################
+        // Public Properties
+        // ##########################################################################################
 
-        public TOptions Get(string name) => _monitor.Get(name);
+        public TOptions Value => _Monitor.CurrentValue;
+
+        public TOptions Get(string name) => _Monitor.Get(name);
+
+        // ##########################################################################################

```

Comment:
Please remove all the // ### comment blocks, not a convention I'd like to add to this repo!